ui-model/src/main/java/com/michaldrabik/ui_model/locale/AppLocale.kt

```diff
@@ -0,0 +1,21 @@
+package com.michaldrabik.ui_model.locale
+
+data class AppLocale(
+  val language: AppLanguage,
+  val country: AppCountry,
+) {
+
+  companion object {
+    const val DELIMITER = "_"
+    fun default() = AppLocale(
```

Comment:
Thinking this should actually be converted into property to avoid creating a new object all the time.